src/tsconfig.json

```diff
@@ -1,12 +1,16 @@
 {
     "compilerOptions": {
+        "target": "es5",
         "noImplicitAny": true,
         "noImplicitReturns": true,
         "noImplicitThis": false,
         "noUnusedLocals": true,
         "noUnusedParameters": true,
         "suppressImplicitAnyIndexErrors": true,
         "strictNullChecks": false,
+        "lib": [
+            "dom", "es2015" // dom for console.log only
```

Comment:
so do you think this could be replaced with:
` "lib": [
  "es2015.core", "scripthost"
]`
?